### DISCUSSION TITLE
error after "brew upgrade"

### DISCUSSION THREAD

**Discussion by Sheldontao** [Homebrew/discussions#3514] [Getting started]
<img width="570" alt="image" src="https://user-images.githubusercontent.com/36765652/183672526-3bd05b16-dd07-4766-acdc-db9eb2b1a367.png">
Error: Failure while executing; `cp -pR /private/tmp/d20220809-50611-95stcj/highlight/. /usr/local/Cellar/highlight` exited with 1. Here's the output:

**Comment by SMillerDev** [comment]
You'll need to fix the permissions of `/usr/local/Cellar/highlight` so your user can write there.

**Reply by Sheldontao** [reply]
<img width="265" alt="image" src="https://user-images.githubusercontent.com/36765652/183676220-d5ad1e33-d56d-4fdb-892e-5caa8130acb6.png">
I though it was right

**Reply by SMillerDev** [reply]
I have no idea how to see it here, but it should be owned by the user running the brew commands.

**Reply by Sheldontao** [reply]
<img width="836" alt="image" src="https://user-images.githubusercontent.com/36765652/183678677-ef6abe26-eba4-4b8c-a6d3-47c2236889b2.png">

**Reply by SMillerDev** [reply]
Try running `ls -al /usr/local/Cellar/highlight` and post the output as text because screenshots are hard to use.

**Reply by Sheldontao** [reply]
total 16
drwxrwxrwx+   4 root          wheel   128 Aug  9 22:50 .
drwxrwxr-x  104 taiyangxiang  admin  3328 Aug  9 22:31 ..
-rw-r--r--@   1 taiyangxiang  wheel  6148 Aug  9 22:45 .DS_Store
drwxrwxr-x    4 root          wheel   128 Feb 11 19:05 3.42

**Reply by SMillerDev** [reply]
Yeah, none of that should be owned by root. You should use `chown` to make sure that your user is the owner of everything under `/usr/local`

**Reply by Sheldontao** [reply]
is here any command line I can just copy? I don't know anything about `chown`

**Reply by Sheldontao** [reply]
`chown [-R] taiyangxiang /usr/local/`
---
right？
